### PR TITLE
Identify the login page explicitly by the page title

### PR DESCRIPTION
**Pull request by Pytal**
By default the page title would be "Login - Nextcloud"

### CODE REVIEW

File: core/src/views/Login.vue [hunk 1]
```diff
@@ -161,6 +161,10 @@ export default {
 		}
 	},
 
+	beforeCreate() {
+		document.title = t('core', 'Login') + ` - ${document.title}`
```
Comment (szaimen) [inline core/src/views/Login.vue:165]: maybe?
```suggestion
		document.title = t('core', 'Login to') + ` ${document.title}`
```
or
```suggestion
		document.title = ` - ${document.title}` + t('core', 'Login')
```
?

Comment (Pytal) [reply core/src/views/Login.vue:165]: What should we go for @nimishavijay?

Comment (nickvergessen) [reply core/src/views/Login.vue:165]: We always prefix, so yeah `Login - previous title` is the way to go

**Comment by Pytal** [comment]
Going with "Login - Nextcloud"

File: core/src/views/Login.vue (hunk 1 above)
Comment (szaimen) [inline core/src/views/Login.vue:165]: All right! lets use an en-dash then:
```suggestion
		document.title = t('core', 'Login') + ` – ${document.title}`
```

**Review by szaimen — APPROVED** [review]
Lgtm but did not test

**Comment by CarlSchwan** [comment]
I feel like this shoukd be done in the backend instead by just passinf a title to the template arguments

**Comment by nickvergessen** [comment]
> I feel like this shoukd be done in the backend instead by just passinf a title to the template arguments

The guests template does not support this yet:
https://github.com/nextcloud/server/blob/master/core/templates/layout.guest.php#L10

but yeah something like
https://github.com/nextcloud/server/blob/master/core/templates/layout.user.php#L19-L24

**Comment by Pytal** [comment]
@CarlSchwan @nickvergessen what do you think now?

**Review by CarlSchwan — APPROVED** [review]

**Review by nickvergessen — APPROVED** [review]

**Comment by blizzz** [comment]
@PVince81 CI was positively failing

**Pull request merged**